docs/unified/novacustom/recovery.md

```diff
@@ -18,13 +18,37 @@ flash chip according to the following documentation sections:
 
 ### BIOS recovery
 
+The procedure will be slightly different depending on the model of your laptop.
```

Comment:
This entire doc will be removed in favor of device specific docs.

Reply:
when? in this MR?

Reply:
We went in a different direction after all (: